articles/azure-maps/map-create.md

```diff
@@ -96,12 +103,12 @@ map.setCamera({
 });
 ```
 
-In the following code,  a [Map object](/javascript/api/azure-maps-control/atlas.map) is constructed via `new atlas.Map()`. Map properties such as `CameraBoundsOptions` can be defined via [setCamera](/javascript/api/azure-maps-control/atlas.map) function of the Map class. Bounds and padding properties are set using `setCamera`.
-
-<br/>
+In the following code, a [Map object](/javascript/api/azure-maps-control/atlas.map) is constructed via `new atlas.Map()`. Map properties such as `CameraBoundsOptions` can be defined via [setCamera](/javascript/api/azure-maps-control/atlas.map) function of the Map class. Bounds and padding properties are set using `setCamera`.
```

Comment:
Do we need to move these links to the bottom of this page?

Reply:
Don't need to, but as time permits I plan to. I'll keep that work separate from this CodePen removal work.